community/odoo/addons

```diff
@@ -0,0 +1 @@
+../../addons/
```

Comment:
What about removing that ?

Reply:
@kmagusiak @rco-odoo Any idea on why this change is needed? Can I safely revert it? It's really annoying to have duplicate in my IDE

Reply:
You can revert it. We may find another solution in the future (related to packaging).

Reply:
Removing in #197562